src/lib/Navigation.svelte

```diff
@@ -10,6 +89,29 @@
 			<div class="mzp-c-navigation-logo">
 				<a href="/">data@mozilla</a>
 			</div>
+			<nav class="mzp-c-menu mzp-is-basic">
+				<ul class="mzp-c-menu-category-list">
+					{#each navigations as navigation}
+						<li class="mzp-c-menu-category mzp-has-drop-down mzp-js-expandable">
+							<a class="mzp-c-menu-title disabled" href="/#" aria-haspopup="true"
+								>{navigation.name}</a
+							>
```

Comment:
This seems like awkward formatting, is this what the linter wanted?

I'd expect something like this:
```js
<a class="..." href="/#" aria-haspopup="true">
  {navigation.name}
</a>
```

Reply:
yup unfortunately our linter wanted it this way 🤷🏻‍♀️ I just tried to fix this manually but it didn't appease Prettier.